standards/request-response.md

```diff
@@ -454,24 +454,24 @@ Access-Control-Allow-Methods: *
 - Custom Headers **MUST** abide by the same rules and guidelines as standard headers.
 - Custom Header names **MUST NOT** be longer than 50 characters. <a name="sps-invalid-custom-header-format" href="#sps-invalid-custom-header-format"><i class="fa fa-check-circle" title="#sps-invalid-custom-header-format"></i></a>
 - Custom Header names **MUST** only contain alpha, numeric, and dash characters: [a-zA-Z0-9-] <a name="sps-invalid-custom-header-format" href="#sps-invalid-custom-header-format"><i class="fa fa-check-circle" title="#sps-invalid-custom-header-format"></i></a>
-- Custom Headers **MUST** start with the prefix `SPS-` (Note: do not prefix with `X-`). <a name="sps-no-x-headers" href="#sps-no-x-headers"><i class="fa fa-check-circle" title="#sps-no-x-headers"></i></a>
+- Custom Headers **MUST** start with the prefix `Sps-` (Note: do not prefix with `X-`). <a name="sps-no-x-headers" href="#sps-no-x-headers"><i class="fa fa-check-circle" title="#sps-no-x-headers"></i></a>
```

Comment:
we should also update the rule `sps-invalid-custom-header-format` to allow for the prefix `Sps` and not `SPS`